### ISSUE TITLE
Feature discussion: making pyscript own debug messages optional through configuration

### ISSUE THREAD

**Issue by wsw70** [custom-components/pyscript#125]
**TL;DR: make `pyscript` own debug messages optional through configuration**

The current `debug` level in logs outputs two distinct information:
- the one, intended, that comes from `log.debug()` in the user's code
- extra verbose information provided by `pyscript` itself

Usually `debug` information is an intentional piece of information that is displayed in specific conditions (during development, but not production for instance). It is specific code inserted by the developer of the application (which in the case of `pyscript` I interpret as the code written by end users such as myself) to provide information when they need to.

It is usually possible to filter out debug information from other modules - I usually use something along the lines of

```
import logging.config
import os

# setup logging
logging.config.dictConfig({
    'formatters': {
        'standard': {
            'format': "%(asctime)s [%(module)s] %(levelname)s %(message)s"
        },
    },
    'handlers': {
        'default': {
            'formatter': 'standard',
            'class': 'logging.StreamHandler',
        },
    },
    'loggers': {
        '': {  # root logger
            'handlers': ['default'],
            'level': logging.ERROR,
            'propagate': False
        },
        'googlecalendar': {
            'handlers': ['default'],
            'level': logging.DEBUG if os.environ.get('DEBUG', '').lower() == 'true' else logging.INFO,
            'propagate': False
        },
    },
    "disable_existing_loggers": True,
    "version": 1,
})
log = logging.getLogger('googlecalendar')
```

This allows me to precisely filter the kind of debug (and above) messages I want to see.

The proposal I would like to discuss is to move the `pyscript` debug messages to an opt-in or opt-out feature, configurable in `configuration.yaml`

**Comment by craigbarratt** [comment]
The logging path for all user code includes the global context name, followed by the function name.  So you can turn on debugging on a per-script file or per-function basis.

So instead of doing this:
```
logger:
  custom_components.pyscript: debug
```
which turns on debugging for both pyscript internals and user code, you could do this:
```
logger:
  custom_components.pyscript.file: debug
  custom_components.pyscript.apps: debug
  custom_components.pyscript.scripts: debug
```
which just turns on debugging for user scripts in the `pyscript` directory, and below the `pyscript/apps` and `pyscript/scripts` directories.

**Comment by wsw70** [comment]
Thank you - this is actually in the docs (https://hacs-pyscript.readthedocs.io/en/latest/reference.html?highlight=log#logging), the point I did not get is that `custom_components.pyscript.file` relates to all the files in the `pyscript` directory, and that `custom_components.pyscript` to the internal logs
